api/lib/infrastructure/repositories/certification-center-membership-repository.js

```diff
@@ -92,7 +92,9 @@ const findActiveByCertificationCenterIdSortedById = async function ({ certificat
       certificationCenterId,
       disabledAt: null,
     })
-    .orderBy('certification-center-memberships.id', 'ASC');
+    .orderByRaw('("role") asc')
```

Comment:
`.orderByRaw('("role") asc')` correspond à une règle métier qui dirait que les utilisateurs doivent doivent être ordonnés alphabétiquement selon leur rôle, alors que la règle métier est que les utilisateurs ayant le rôle `ADMIN` doit être listés en premier. Aussi voici une suggestion pour avoir un ordonnancement qui correspond à la règle métier :  

```suggestion
    .orderByRaw("CASE role WHEN 'ADMIN' THEN 1 ELSE 2 END")
```

Reply:
Oui tout à fait ! Merci 🙂 c'est corrigé dans ce commit 75ae72cccd6222bc240f2c9d1dd4389f80c59d28